docs/cne_install.md

```diff
@@ -0,0 +1,189 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+The CNE installer can be used only for upgrades and not for fresh installations.
+See [Install the System Admin Toolkit Product Stream](install.md) for
+installation instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both
+automated and logged to help you save time. The CNE installer can be used to
+upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX
+System Software Getting Started Guide (S-8000)*](<https://www.hpe.com/support/ex-S-8000>)
+for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
+- There must be at least 2 gigabytes of free space on the manager NCN on which
+the procedure is run.
+
+### Notes on the Procedures
+
+- Ellipses (`...`) in shell output indicate omitted lines.
+- In the examples below, replace `x.y.z` with the version of the SAT product stream
+being upgraded.
```

Comment:
I guess this is not part of the style/lint checks so I am hesitant to even mention it, but I would slightly prefer if we indented continuation lines to match the level of text above it, e.g.

```suggestion
- In the examples below, replace `x.y.z` with the version of the SAT product stream
   being upgraded.
```

This also would apply to lines: 53, 119 and lines 21-22 of `install.md`.

Sorry I didn't bring this up before, from my perspective it would be nice to do for consistency but the style check doesn't seem to have an opinion about it.

Reply:
Apologies, I didn't realize this was the style elsewhere in the repo (one of the problems of having four repos with different styles). I updated as requested, but let me know if I missed any.